layers/+completion/helm/packages.el

```diff
@@ -117,7 +117,17 @@
                 (lambda ()
                   (unless (configuration-layer/package-usedp 'smex)
                     (spacemacs/set-leader-keys
-                      dotspacemacs-emacs-command-key 'helm-M-x)))))
+                      dotspacemacs-emacs-command-key 'helm-M-x))))
+
+      ;; In follow mode, follow on keystrokes as well as up/down
+      (setq helm-follow-follow-on-update t)
+      ;; Run any search with follow mode on
+      (defun spacemacs//helm-do-search-with-follow (helm-source helm-command)
+        (let ((prev-follow-val (helm-attr 'follow (symbol-value helm-source))))
+          (helm-attrset 'follow 1 (symbol-value helm-source))
+          (call-interactively helm-command)
```

Comment:
Is there a reason to prefer `call-interactively` over `funcall`? If the intention is to change as little of the behavior as possible, then:
```elisp
(defun do-foo ()
  (interactive)
  (funcall 'foo))
```
Is closer to:
```elisp
(defun do-foo ()
  (interactive)
  (foo))
```
than:
```elisp
(defun do-foo ()
  (interactive)
  (call-interactively 'foo))
```